source_dir/foo.tf

```diff
@@ -0,0 +1,46 @@
+resource "yandex_mdb_postgresql_cluster" "ex" {
```

Comment:
I think it would be also useful to add a simple bash test with converting only one file and comparing the result with `diff`.